CONTRIBUTING.rst

```diff
@@ -18,12 +18,37 @@ first in an appropriate venue, drafting a PEP and gathering feedback, and
 developing at least a prototype reference implementation of your idea.
 
 
+Contributing changes to existing PEPs
+-------------------------------------
+
+In general, most non-Draft/Active PEPs are considered to be historical
+documents rather than living specifications or documentation. Major changes to
+their core content usually require a new PEP, while smaller modifications may
+or may not be appropriate, depending on the PEP's status. See `PEP 1
+<https://www.python.org/dev/peps/pep-0001/#pep-maintenance>`__ for more.
```

Comment:
I understand this is just a summary, but there's such a big divide between Draft and Active PEPs (SC approval comes to mind :-) that it's wrong to describe them as more or less the change here.

Reply:
(Sorry, I confused Active with Accepted. Never mind then.)